tox.ini

```diff
@@ -1,13 +1,22 @@
 [tox]
 envlist = py27
 
+[pylama]
+format = mccabe,pep8,pep257,pyflakes,pylint
+ignore = C901
+
+[pylama:pep8]
+max_line_length = 100
+
 [testenv:py27]
 deps =
     mock
     mozfile >= 1.1
     mozprocess >= 0.22
     pytest
+    pylama
     wptserve >= 1.4.0
 
 commands =
+    pylama mozdownload
     py.test
```

Comment:
So I asked in #pylib for feedback in how to use pylama and the suggestion was to keep it separate. So this will be fine.